spec/factories/doorkeeper.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :oauth_application, class: Doorkeeper::Application do
+    name { "Application" }
```

Comment:
Do we also need a seed for something like this in `seeds.rb` so we get this for free when we spin up a new development DB?

Reply:
No, these are just the FactoryBot defaults for tests

Reply:
Were you able to test this locally using curl and govuk docker?
Or am I missing something? I thought for setups like this we'd need new entries [like these](https://github.com/alphagov/govuk-account-manager-prototype/blob/main/db/seeds.rb#L20-L24) to test it on dev?

Reply:
These are just defaults for `FactoryBot.create` calls, we don't need an application in the database with the name `Application` - much like we don't seed the database with an `email@example.com` user: https://github.com/alphagov/govuk-account-manager-prototype/blob/main/spec/factories/users.rb#L3